src/main/java/org/apache/datasketches/sampling/ReservoirItemsSketch.java

```diff
@@ -259,7 +259,7 @@ public static <T> ReservoirItemsSketch<T> heapify(final Memory srcMem,
 
     final int itemsToRead = (int) Math.min(k, itemsSeen);
     final T[] data = serDe.deserializeFromMemory(
-        srcMem.region(preLongBytes, srcMem.getCapacity() - preLongBytes), itemsToRead);
+        srcMem.region(preLongBytes, srcMem.getCapacity() - preLongBytes), 0, itemsToRead);
```

Comment:
It is not clear to me why this region is needed here. this new offset parameter can be used instead. or, the other way around, why is this new offset parameter needed, if we always can use a region?

Reply:
You are correct.  My modification only made the error go away, I should have replaced the use of region entirely.  Fixed.